README.md

```diff
@@ -0,0 +1,33 @@
+# Core Snap
```

Comment:
`# The core snap`

Let's please respect the capitalization we generally use for the word "core".